src/Ast/Expression.elm

```diff
@@ -38,7 +38,7 @@ type Expression
   | Record (List (Name, Expression))
   | RecordUpdate Name (List (Name, Expression))
   | If Expression Expression Expression
-  | Let (List (Name, Expression)) Expression
+  | Let (List (Name, List Name, Expression)) Expression
```

Comment:
Given that it changes type declaration it would be considered a major change, requiring all of the relying libraries to change their implementations.

Is there maybe any other way to express that without changing the type anotation?